tests/integration/config/log4j.properties

```diff
@@ -0,0 +1,71 @@
+# Unspecified loggers and loggers with additivity=true output to server.log
```

Comment:
This is the default logging configuration, the only change is that stdout is no longer used.